### PR TITLE
Add tool test for what happens when output filter throws exception.

### PR DESCRIPTION
**Pull request by jmchilton**
Run new test with:

```
./run_tests.sh -framework -id output_filter_exception_1
```

xref https://github.com/galaxyproject/tools-iuc/pull/883
xref https://github.com/galaxyproject/planemo/commit/a066afb1c2c30f4ef2c3033ab9a884b31c06e783
xref https://github.com/galaxy-iuc/standards/pull/31

### CODE REVIEW

File: test/functional/tools/output_filter_exception_1.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <when value="advanced">
+        <param falsevalue="" truevalue="--adv_opt1" help="" name="adv_opt1" type="boolean" />
+      </when>
+    </conditional>
+  </inputs>
+  <outputs>
+    <data format="txt" from_work_dir="1" name="out_1">
+    </data>
+    <data format="txt" from_work_dir="2" name="out_2">
+      <filter>opitons['adv_opt1'] is True</filter>
```
Comment (nsoranzo) [inline test/functional/tools/output_filter_exception_1.xml]: s/opitons/options/

Comment (jmchilton) [inline test/functional/tools/output_filter_exception_1.xml]: Fixed, good catch!

**Comment by nsoranzo** [comment]
:+1: Thanks @jmchilton!

**Comment by nsoranzo** [comment]
Framework tests pass locally, merging.

**Comment by jmchilton** [comment]
Thanks @nsoranzo!

**Pull request merged**